oc_cdtapi/OnecAPI.py

```
@@ -0,0 +1,229 @@
+                    f"Components request failed with status {response.status_code} "
+                    f"for endpoint: {self.root}/components?client={client_code}"
+                )
+                logging.error(error_msg)
+                return None, error_msg
+                
+        except HttpAPIError as e:
+            error_msg = f"Error communicating with 1C API: {e}"
+            logging.error(error_msg)
+            return None, error_msg
+    
+    def _build_query_string(self, params):
```

Ruff (0.15.4)

[warning] Missing return type annotation for private function `_build_query_string`

(ANN202)